### PR TITLE
debug test zqd server startup

### PR DESCRIPTION
**Pull request by alfred-landrum**

### CODE REVIEW

**Comment by mattnibs** [comment]
@alfred-landrum seems like this is a difficult to reproduce bug. I think we should merge the changes you made to services.sh.

**Comment by alfred-landrum** [comment]
I did finally get a run here with the failure, but the output doesn't give any good clues:
https://github.com/brimsec/zq/runs/1246114767#step:11:985
```
        --- expected
        +++ actual
        @@ -1,6 +1,14 @@
        -testsp: space created
        -===
        -testsp: space removed
        -===
        -no spaces exist
        +timed out waiting for file "/tmp/tmp.T3mH8Hfk7P/zqd" to appear
        +Endpoint:  http://localhost:44519
         
        +Browser Access:
        +   http://localhost:44519
        +
        +Object API (Amazon S3 compatible):
        +   Go:         https://docs.min.io/docs/golang-client-quickstart-guide
        +   Java:       https://docs.min.io/docs/java-client-quickstart-guide
        +   Python:     https://docs.min.io/docs/python-client-quickstart-guide
        +   JavaScript: https://docs.min.io/docs/javascript-client-quickstart-guide
        +   .NET:       https://docs.min.io/docs/dotnet-client-quickstart-guide
        +Detected default credentials 'minioadmin:minioadmin', please change the credentials immediately using 'MINIO_ACCESS_KEY' and 'MINIO_SECRET_KEY'
        +
```

File: ztests/suite/zqd/services.sh [hunk 1]
```diff
@@ -33,7 +35,7 @@ export AWS_ACCESS_KEY_ID=minioadmin
 export AWS_SECRET_ACCESS_KEY=minioadmin
 export AWS_S3_ENDPOINT=http://localhost:$(cat $portdir/minio)
 
-zqd listen -l=localhost:0 -portfile="$portdir/zqd" -data="$zqdroot" -loglevel=warn &
+zqd listen -l=localhost:0 -portfile="$portdir/zqd" -data="$zqdroot" -loglevel=warn > zqd.log 2>&1 &
```
Comment (mattnibs) [inline ztests/suite/zqd/services.sh:38]: Maybe also drop `loglevel=warn` so we get more data?
```suggestion
zqd listen -l=localhost:0 -portfile="$portdir/zqd" -data="$zqdroot" > zqd.log 2>&1 &
```

File: Makefile [hunk 2]
```diff
@@ -69,7 +69,7 @@ test-unit:
 	@go test -short ./...
 
 test-system: build bin/minio bin/$(ZEEKPATH) bin/$(SURICATAPATH)
-	@ZTEST_PATH=$(CURDIR)/dist:$(CURDIR)/bin:$(CURDIR)/bin/$(ZEEKPATH):$(CURDIR)/bin/$(SURICATAPATH) go test -v .
+	@ZTEST_PATH=$(CURDIR)/dist:$(CURDIR)/bin:$(CURDIR)/bin/$(ZEEKPATH):$(CURDIR)/bin/$(SURICATAPATH) go test -v -count 1000 -timeout 0 -run TestZq/ztests/suite/zqd/s3/search
```
Comment (mattnibs) [inline Makefile:72]:
```suggestion
	@ZTEST_PATH=$(CURDIR)/dist:$(CURDIR)/bin:$(CURDIR)/bin/$(ZEEKPATH):$(CURDIR)/bin/$(SURICATAPATH) go test -v -count 1000 -timeout 0 -run TestZq/ztests/suite/zqd/s3/search .
```

**Comment by mattnibs** [comment]
Another failure here: https://github.com/brimsec/zq/runs/1251000488?check_suite_focus=true

File: cmd/zqd/listen/command.go [hunk 3]
```diff
@@ -131,9 +132,11 @@ func (c *Command) Run(args []string) error {
 	}()
 	srv := httpd.New(c.listenAddr, h)
 	srv.SetLogger(c.logger.Named("httpd"))
+	fmt.Println("server.starting")
```
Comment (mattnibs) [inline cmd/zqd/listen/command.go:135]:
```suggestion
```

**Comment by mattnibs** [comment]
Closing and creating a pr for this.

**Pull request closed without merging**